gninasrc/lib/flexinfo.cpp

```diff
@@ -294,3 +285,20 @@ void FlexInfo::extractFlex(OpenBabel::OBMol& receptor, OpenBabel::OBMol& rigid,
   rigid.EndModify();
 
 }
+
+void FlexInfo::printFlex() const{
+
+  // Residues are stored as unordered_set
+  // Sort before printing
+  std::vector<std::tuple<char, int, char> > sortedres(residues.begin(), residues.end());
+  sort(sortedres.begin(), sortedres.end());
+
+  if (sortedres.size() > 0) {
+    log << "Flexible residues:";
+    for (unsigned i = 0, n = sortedres.size(); i < n; i++) {
+      log << " " << get<0>(sortedres[i]) << ":" << get<1>(sortedres[i]);
+      if(get<2>(sortedres[i]) > 0) log << ":" << get<2>(sortedres[i]);
+    }
+    log << "\n";
+  }
+}
```

Comment:
what's up with these null terminators

Reply:
I think it's just GitHub not working properly... The full change is the following:
```c++
@@ -294,3 +285,20 @@ void FlexInfo::extractFlex(OpenBabel::OBMol& receptor, OpenBabel::OBMol& rigid,
  rigid.EndModify();

}

void FlexInfo::printFlex() const{

  // Residues are stored as unordered_set
  // Sort before printing
  std::vector<std::tuple<char, int, char> > sortedres(residues.begin(), residues.end());
  sort(sortedres.begin(), sortedres.end());

  if (sortedres.size() > 0) {
    log << "Flexible residues:";
    for (unsigned i = 0, n = sortedres.size(); i < n; i++) {
      log << " " << get<0>(sortedres[i]) << ":" << get<1>(sortedres[i]);
      if(get<2>(sortedres[i]) > 0) log << ":" << get<2>(sortedres[i]);
    }
    log << "\n";
  }
} 
```